src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/StopPoints/Table/index.tsx

```diff
@@ -17,7 +18,7 @@ import messages from './messages';
 import './styles.scss';
 
 type Props = {
-  stopPoints: any[];
+  stopPoints: StopPoint[];
   onRowClick: (i: number) => void;
   onDeleteClick: (sp: any) => void;
```

Comment:
```suggestion
  onDeleteClick: (sp: StopPoint) => void;
```

Reply:
Detta fungerar ikke, argumentet som sänds in är `number | null`. Men ska se på det =)